src/Spice86.Core/Emulator/CPU/CPU.cs

```diff
@@ -1345,6 +1371,9 @@ private static bool IsStringOpUpdatingFlags(int stringOpCode)
             case 0x66:
                 _instructions16Or32 = _instructions32;
                 break;
+            case 0x67:
+                AddressSize = 32;
```

Comment:
Shouldn't this be done in a public method so low-level C# overrides of ASM code can call this instruction directly ?